contracts/src/v0.8/KeeperRegistry.sol

```diff
@@ -112,6 +121,12 @@ contract KeeperRegistry is
     BIDIRECTIONAL
   }
 
+  enum PaymentModel {
+    L1,
```

Comment:
nit. maybe call it default, some L2s might not have this complexity

Reply:
Done